marvel_superheroes/src/components/HomePage/HomePage.tsx

```diff
@@ -7,7 +7,9 @@ import queryString from 'query-string';
 import { RouteComponentProps } from "react-router";
 import HeroSearchBar from "../HeroSearchBar/HeroSearchBar";
 import PaginationRounded from '../PaginationRounded';
-import { getHeroes } from "../api";
+import { connect } from 'react-redux'
+import { fetchHeroes, fetchHeroesSaga, hideLoader, IFetchHeroes, IFetchHeroesSaga, ILoader, ISetHeroesCurrentPage, ISetHeroesNameStartWith, ISetHeroesOrderBy, setHeroesCurrentPage, setHeroesNameStartWith, setHeroesOrderBy, showLoader } from "../../store/actions/actions";
+import { store } from "../../index";
```

Comment:
also see that some strings are double quote  and others are single. Add appropriate eslint rule to handle this